projects/rich-previews/src/google-product-desktop/google-product-desktop.css

```diff
@@ -0,0 +1,260 @@
+@import "../../@shared/css/defaults.css";
+@import "../../@shared/css/google.css";
+
+.scroll-container--fake {
+  max-width: 70.25rem;
+}
+
+/* Product */
+.product {
+  background-color: var(--color-white);
+  border-radius: 0.5rem;
+  box-shadow: 0 1px 6px 0 rgba(0, 0, 0, 0.16);
+  max-width: 70.25rem;
+  min-width: 38.875rem;
+  padding: 0.75rem;
+}
+
+.product--no-hover {
+  cursor: default;
+}
+
+.product__image,
+.product__merchant-name,
+.product__merchant-review,
+.product__name,
+.product__review {
+  cursor: pointer;
+}
+
+.product__merchant-name:hover,
+.product__merchant-review:hover,
+.product__name:hover,
+.product__review:hover,
+.product-carrousel__item:hover .product__name {
+  text-decoration: underline;
+}
+
+.product__image {
+  display: block;
+  height: 11.25rem;
+  margin: 0.75rem 0.75rem 0.75rem 0;
+  object-fit: contain;
+  object-position: center;
+  width: 11.25rem;
+}
+
+.product__content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 0.75rem;
+}
+
+.product__merchant-name,
+.product__merchant-review,
+.product__price {
+  line-height: 1.75rem;
+  margin-right: 0.375rem;
+}
+
+.product__name {
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 2;
+  color: var(--color-shuttle-gray);
+  display: -webkit-box;
+  font-size: 1.25rem;
+  line-height: 1.3;
+  margin-bottom: 0.375rem;
+  overflow: hidden;
+}
+
+.product__price {
+  color: var(--color-mine-shaft);
+  font-size: 1.125rem;
+  font-weight: 700;
+}
+
+.product__description,
+.product__merchant-name,
+.product__merchant-review,
+.product__review {
+  font-size: 0.875rem;
+}
+
+.product__description {
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 1;
+  display: -webkit-box;
+  overflow: hidden;
+}
+
+.product__merchant-name {
+  color: var(--color-royal-blue);
+}
+
+.product__merchant-review {
+  color: var(--color-boulder);
+  white-space: nowrap;
+}
+
+.merchant-review__score {
+  color: var(--color-la-palma);
+}
+
+.product__description,
+.product__review {
+  color: var(--color-shuttle-gray);
+  line-height: 1.125rem;
+  margin-top: 0.375rem;
+}
+
+.review__stars-1,
+.review__stars-2,
+.review__stars-3,
+.review__stars-4,
+.review__stars-5,
+.review__stars-wrapper {
+  background-image: url(../@assets/google-shopping-sprites.webp);
```

Comment:
No fallback when `.webp` images are not supported?

Reply:
Vervangen met een png, net als de andere images.